### PR TITLE
Add documentation structure changes

### PR DESCRIPTION
**Pull request by SanduDS**
## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: outlookmail/Package.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-### Package overview
+## Package overview
 The microsoft.outlook.mail is a [Ballerina](https://ballerina.io/) connector for Microsoft Outlook mail service.
 This package provides the capability to access Microsoft Outlook mail service.
-#### Compatibility
+### Compatibility
 |                               | Version                              |
 |-------------------------------|--------------------------------------|
 | Ballerina Language            | **Ballerina Swan Lake Beta 2**       |
 | Outlook.mail API              | **Microsoft Graph (Mail) API v1.0**  |
```
Comment (LakshanSS) [inline outlookmail/Package.md:9]: Remove ** from versions

File: outlookmail/Module.md [hunk 2]
```diff
@@ -1,29 +1,28 @@
 ## Overview
-Ballerina connector for Microsoft Outlook mail provides access to Microsoft Outlook mail service in Microsoft Graph v1.0 via Ballerina language easily. It provides capability to perform more useful functionalities provided in Microsoft outlook mail such as sending messages, listing messages, creating drafts, mail folders, deleting and updating messages etc. 
+Microsoft Outlook mail API provides capability to access Outlook mail operations related to messages, attachments, drafts,
```
Comment (abeykoon) [inline outlookmail/Module.md:2]: I like the earlier description :)

File: README.md [hunk 3]
```diff
@@ -7,7 +7,7 @@ Ballerina connector for Microsoft Outlook [Mail](https://docs.microsoft.com/en-u
 access Microsoft Cloud service resources. This version of the connector supports most of the important mail service related operations.
  
 For more information, go to the module(s).
-- [ballerinax/microsoft.outlook.mail](https://github.com/SanduDS/module-ballerinax-microsoft.outlook.mail/tree/main/outlookmail)
+- [microsoft.outlook.mail](outlookmail/Module.md)
```
Comment (shaniR) [inline README.md:10]: Let's rephrase the above paragraph as "Ballerina connector for Microsoft Outlook [Mail](https://docs.microsoft.com/en-us/graph/api/resources/mail-api-overview?view=graph-rest-1.0) provides you the capability to access Microsoft Cloud service resources. The connector internally uses the Microsoft Graph that provides REST web APIs."

Comment (shaniR) [reply README.md:10]: Let's rephrase the above sentence as : "This version of the connector supports most of the mail service-related operations."

File: outlookmail/Module.md (hunk 2 above)
Comment (shaniR) [reply outlookmail/Module.md:2]: Shall we rephrase this to: Ballerina connector for Microsoft Outlook mail provides access to the Microsoft Outlook mail service in Microsoft Graph v1.0 via the [Ballerina language](https://ballerina.io/). It provides the capability to perform more useful functionalities provided in Microsoft outlook mail such as sending messages, listing messages, creating drafts, mail folders, deleting messages, updating messages, etc.

File: outlookmail/Module.md [hunk 4]
```diff
@@ -1,29 +1,28 @@
...
+This module supports [Microsoft Graph (Mail) API v1.0](https://docs.microsoft.com/en-us/graph/api/resources/message?view=graph-rest-1.0).
 
 ## Prerequisites
 Before using this connector in your Ballerina application, complete the following:
 * Create [Microsoft Outlook Account](https://outlook.live.com/owa/)
 * Obtaining tokens
-        
-    Follow [this link](https://docs.microsoft.com/en-us/graph/auth-v2-user#authentication-and-authorization-steps) and obtain the client ID, client secret and refresh token.
-* Configure the connector with obtained tokens
+1. Follow [this link](https://docs.microsoft.com/en-us/graph/auth-v2-user#authentication-and-authorization-steps) and obtain the client ID, client secret and refresh token.
```
Comment (shaniR) [inline outlookmail/Module.md:11]: client ID, client secret, and refresh token.

File: outlookmail/Package.md [hunk 5]
```diff
@@ -1,16 +1,16 @@
 Connects to Microsoft Outlook Mail from Ballerina
-### Package overview
+## Package overview
 The microsoft.outlook.mail is a [Ballerina](https://ballerina.io/) connector for Microsoft Outlook mail service.
```
Comment (shaniR) [inline outlookmail/Package.md:3]: Let's use the connector name?

**Review by shaniR — CHANGES_REQUESTED** [review]

**Review by abeykoon — APPROVED** [review]

File: outlookmail/client.bal [hunk 6]
```diff
@@ -17,22 +17,26 @@
 import ballerina/http;
 import ballerina/io;
 
-# Ballerina Client for microsoft outlook mail operations.
+# Microsoft Outlook mail API provides capability to access Outlook mail operations related to messages, attachments, drafts,
```
Comment (shaniR) [inline outlookmail/client.bal:20]: provides the

File: outlookmail/client.bal [hunk 7]
```diff
@@ -17,22 +17,26 @@
...
 }
-public client class Client {
-    http:Client httpClient;
-    Configuration config;
+public isolated  client class Client {
+    final http:Client httpClient;
+    final readonly & Configuration config;
 
-    # Client Initialization
+    # Gets invoked to initialize the `connector`.
```
Comment (shaniR) [inline outlookmail/client.bal:30]: Initializes the connector.
